sdk/identity/tests.yml

```diff
@@ -3,8 +3,20 @@ trigger: none
 extends:
   template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
   parameters:
-    ServiceDirectory: identity
+    ServiceDirectory: identity    
+    PreSteps:
+      - pwsh: |
+          [System.Convert]::FromBase64String($env:PFX_CONTENTS) | Set-Content -Path $(Agent.TempDirectory)/test.pfx -Encoding Byte
+          Set-Content -Path $(Agent.TempDirectory)/test.pfx -Value $env:PEM_CONTENTS
```

Comment:
Isn't this overwriting the test.pfx file that already exists?

Reply:
Yes, this should write to test.pem. Updated.